src/shared/components/event-start/event-start-component.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ComponentPortalExample1Component, ComponentPortalExample2Component, EventStartComponent } from './event-start.component';
+import { PortalModule } from '@angular/cdk/portal';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    PortalModule
+  ],
+  declarations: [
+    EventStartComponent,
+    ComponentPortalExample1Component,
+    ComponentPortalExample2Component
+  ],
+  entryComponents: [
+    EventStartComponent,
+    ComponentPortalExample1Component,
+    ComponentPortalExample2Component
+  ],
+  exports: [
+    EventStartComponent,
+    ComponentPortalExample1Component,
```

Comment:
Unless the components for the individual steps that are rendered within the portal are to be used outside of this module as well, there is no need to export them.

Reply:
Point noted and removed it now